### ISSUE TITLE
the possibility to upscale a input image 

### ISSUE THREAD

**Issue by radiorauschen** [divamgupta/diffusionbee-stable-diffusion-ui#238]
sometimes my input image is very small in pixel density and it would be nice to have the possibility to upscale that image as well.

(it also would be nice to be able upload and upscale older generated images.)

thanks!

**Comment by drpicox** [comment]
Do you mean this?
Did you update to latest version?

![Screenshot 2022-10-21 at 19 18 35](https://user-images.githubusercontent.com/2758345/197252739-ab4d1b54-82c6-4a66-a173-f79533f2bb96.png)

**Comment by enzyme69** [comment]
It's a hack, but maybe do your input image2image, then set Strength to maximum (original image as outcome), then use the output result and upscale image, then you use that as input.

**Comment by radiorauschen** [comment]
> Do you mean this? Did you update to latest version?
> 
> ![Screenshot 2022-10-21 at 19 18 35](https://user-images.githubusercontent.com/2758345/197252739-ab4d1b54-82c6-4a66-a173-f79533f2bb96.png)

no, i was talking about the input image. as far as i can see you can only use upscale on images that were generated and not the source image itself.

**Comment by xcrap** [comment]
This is the best free upscale I use, amazing results, it would be good to know if this would be possible to implement.

https://colab.research.google.com/drive/19euI_7GAgbvMoZsuPj9SZseDeuFnwBj8#scrollTo=OwGaO4w8asJN
